pootle/apps/pootle_project/models.py

```diff
@@ -142,7 +142,7 @@ class ProjectURLMixin(object):
     """
 
     def get_absolute_url(self):
-        lang, proj, dir, fn = split_pootle_path(self.pootle_path)
+        __, proj, __, __ = split_pootle_path(self.pootle_path)
```

Comment:
I took the decision that `split_pootle_path` is pretty well understood and thus `lang_` clouds the issue and is not useful.  What this does show though is that we probably want helper functions that actually return `proj` from `self.pootle_path`